src/pages/landing-pages/[landingPage].tsx

```diff
@@ -0,0 +1,138 @@
+import { InferGetServerSidePropsType, GetServerSideProps } from 'next'
+import { GridContainer, Grid, Accordion } from '@trussworks/react-uswds'
+import Link from 'next/link'
+import { client } from 'lib/keystoneClient'
+import { useUser } from 'hooks/useUser'
+import Loader from 'components/Loader/Loader'
+import { withLandingPageLayout } from 'layout/DefaultLayout/LandingPageLayout'
+import { GET_LANDING_PAGE } from 'operations/cms/queries/getLandingPage'
+import { ArticleList } from 'components/ArticleList/ArticleList'
+import Collection from 'components/Collection/Collection'
+import Bookmark from 'components/Bookmark/Bookmark'
+import {
+  DocumentType,
+  DocumentSectionType,
+  Collection as CollectionType,
+} from 'types'
+import { isPdf, handleOpenPdfLink } from 'helpers/openDocumentLink'
+import styles from 'styles/pages/landingPages.module.scss'
+
+const LandingPage = ({
+  pageTitle,
+  pageDescription,
+  articles,
+  collections,
+  documents,
+}: InferGetServerSidePropsType<typeof getServerSideProps>) => {
+  const { user } = useUser()
+
+  return (
+    <GridContainer>
+      {!user ? (
+        <Loader />
+      ) : (
+        <div>
+          <h1>{pageTitle}</h1>
+          <p>{pageDescription}</p>
+          <h2>Documentation</h2>
+          <Grid row gap="lg">
+            {documents &&
+              documents.map((s: DocumentSectionType, i: number) => (
+                <Grid col={12} key={s.id} className={styles.accordionGrid}>
+                  <Accordion
+                    className={styles.accordion}
+                    bordered
+                    items={[
+                      {
+                        title: s.title,
+                        expanded: i === 0 ? true : false,
```

Comment:
I'm wondering if we need to remove this line? Not sure if we want to automatically expand the first section here like we do on `/ussf-documentation`. Maybe a question for design?

Reply:
agree -- i personally think all sections should be expanded on page load but that's just me 😄